web/angular-wallet/package.json

```diff
@@ -86,8 +86,8 @@
     "zone.js": "0.9.1"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "^0.800.6",
-    "@angular/cli": "^8.0.1",
+    "@angular-devkit/build-angular": "^0.800.3",
+    "@angular/cli": "^8.0.3",
```

Comment:
my local build was borked without this, see https://stackoverflow.com/questions/52038332/cannot-read-property-ngmetadataname-of-undefined